stable/rabbitmq/templates/pdb.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.podDisruptionBudget -}}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "rabbitmq.fullname" . }}
+  labels:
+    app: {{ template "rabbitmq.name" . }}
+    chart: {{ template "rabbitmq.chart" .  }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "rabbitmq.name" . }}
+      release: "{{ .Release.Name }}"
+{{ toYaml .Values.podDisruptionBudget | indent 2 }}
```

Comment:
Wouldn't it make sense to template this as well?

Reply:
Thanks for your comment. 
Do you mean "template" the `matchLabels` section? 
I didn't update that part because all the Chart is using this synthax.

About the `{{ toYaml .Values.podDisruptionBudget | indent 2 }}` i don't think we need to template that part because the expected attributes here will be only some integers.
Do you have a specific use - case in mind that need to template this too ?
Let me know, i will adjust that part !

Reply:
Well, I think that we can leave this for later improvements or see if there are actual requirements for this.